routes/repo/download.go

```
@@ -22,6 +22,11 @@ func ServeData(c *context.Context, name string, reader io.Reader) error {
 		buf = buf[:n]
 	}
 
+	commit, err := c.Repo.Commit.GetCommitByPath(c.Repo.TreePath)
+	if err == nil {
+		c.Resp.Header().Set("Last-Modified", commit.Committer.When.Format(http.TimeFormat))
```

Error should be logged, this swallows the error.